v21.2/restore.md

```diff
@@ -63,6 +63,7 @@ You can include the following options as key-value pairs in the `kv_option_list`
 <a name="skip_missing_sequences"></a>`skip_missing_sequences`       | N/A                                         | Use to ignore [sequence](show-sequences.html) dependencies (i.e., the `DEFAULT` expression that uses the sequence).<br><br>Example: `WITH skip_missing_sequences`
 `skip_missing_sequence_owners`                                      | N/A                                         | Must be used when restoring either a table that was previously a [sequence owner](create-sequence.html#owned-by) or a sequence that was previously owned by a table.<br><br>Example: `WITH skip_missing_sequence_owners`
 `skip_missing_views`                                                | N/A                                         | Use to skip restoring [views](views.html) that cannot be restored because their dependencies are not being restored at the same time.<br><br>Example: `WITH skip_missing_views`
+<a name="skip-localities-check"></a>`skip_localities_check`         | N/A                                         | <span class="version-tag">New in v21.2:</span> Use to skip checking localities of a cluster before a restore when there are mismatched regions between the backup's cluster and the target cluster. <br><br>Example: `WITH skip_localities_check`
```

Comment:
I'm confused is this referring to [cluster regions](https://www.cockroachlabs.com/docs/dev/multiregion-overview.html#cluster-regions)?

I realize this is not the first place we overload "locality" vs "region" of course

could be mitigated by updating slightly to "mismatched _cluster_ regions between the backup's cluster ..." and making 'cluster regions' a link to https://www.cockroachlabs.com/docs/dev/multiregion-overview.html#cluster-regions